src/pybind/mgr/dashboard/frontend/src/app/shared/components/date-time-picker/date-time-picker.component.ts

```diff
@@ -59,8 +60,11 @@ export class DateTimePickerComponent implements OnInit {
     if (!mom.isValid() || mom.isBefore(moment())) {
       mom = moment();
     }
-
-    this.date.push(mom.format('YYYY-MM-DD'));
+    if (this.defaultDate === true) {
+      this.date.push([]);
+    } else {
+      this.date.push(mom.format('YYYY-MM-DD'));
+    }
```

Comment:
shouldn't we reverse the logic here?

if defaultDate then set today's set; else leave it empty.

Reply:
modified the code